Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -149,11 +149,6 @@
 @property (nonatomic, strong) BUYShippingRate *shippingRate;
 
 /**
- *  Shipping rate identifier
- */
-@property (nonatomic, readonly) NSString *shippingRateId;
```

Comment:
In order to maintain backward compatibility, and avoid including the `shippingRateId` in the payload - all we need to remove is the will/didChangeValueForKey in setShippingRateId